### PR TITLE
Fix Tile Render using Hybrid with Restir

### PR DESCRIPTION
**Pull request by DMotorygin**
JIRA TICKET: RPRMAYA-3394

PURPOSE: Currently Restir feature if used together with tile render causes HybridPro to produce "checkerboard"-like pattern in rendered tiles.

EFFECT FOR THE USER: UI scripts updated so that if tile render is enabled than Restir is set to "disabled" and can not be enabled.

NOTE FOR REVIEWERS: In order to make Restir work with tiles we would need to use extraordinary amount of video memory. While such fix is possible on test scenes with small tiles, it is not possible for real use cases of very large images being rendered by tile renderer (task for which tile renderer exists for int the first place)

### CODE REVIEW

**Comment by DavidSpeicher** [comment]
Approved

File: FireRender.Maya.Src/scripts/RenderQualityTab.mel [hunk 1]
```diff
@@ -235,7 +243,9 @@ proc createViewportRenderQualityPart()
 			attrControlGrp
 			    //-label "Reservoir Sampling"
```
Comment (Vsevolod1983) [inline FireRender.Maya.Src/scripts/RenderQualityTab.mel:244]: remove comment ?

File: FireRender.Maya.Src/scripts/RenderQualityTab.mel [hunk 2]
```diff
@@ -137,7 +140,9 @@ proc createFinalRenderQualityPart()
 	    attrControlGrp
 	        //-label "Reservoir Sampling"
```
Comment (Vsevolod1983) [inline FireRender.Maya.Src/scripts/RenderQualityTab.mel:141]: remove comment ?

**Review by Vsevolod1983 — CHANGES_REQUESTED** [review]
remove some comments

**Review by Vsevolod1983 — APPROVED** [review]

**Pull request merged**